torchtext/prototype/models/t5/modules.py

```diff
@@ -805,11 +805,13 @@ def __init__(
         layer_norm_eps: float = 1e-6,
         relative_attention_num_buckets: int = 32,
         relative_attention_max_distance: int = 128,
+        token_embeddings: Optional[nn.Module] = None,
```

Comment:
I don't think this should mess up the backprop for the entire model, but can you confirm @Nayef211 ?